src/VirtualClient/VirtualClient.Actions/Sysbench/SysbenchClientExecutor.cs

```diff
@@ -169,7 +173,24 @@ private Task ExecuteWorkloadAsync(EventContext telemetryContext, CancellationTok
                 {
                     if (this.Benchmark == BenchmarkName.OLTP)
                     {
-                        await this.RunOLTPWorkloadAsync(telemetryContext, cancellationToken);
+                        if (this.Action == ClientAction.TruncateDatabase)
+                        {
+                            DependencyPath workloadPackage = await this.GetPackageAsync(this.PackageName, cancellationToken).ConfigureAwait(false);
```

Comment:
Are you using this?

Reply:
Yes, It is being used in a custom profile. Although that can be added to our generic profile for mysql as well with some changes.